introduction/deprecated_features/README.md

```diff
@@ -0,0 +1,22 @@
+# Deprecated Features
+
+This page lists deprecated features in CSM. They are no longer being actively developed and are planned to be removed in a future CSM release.
+This groups the deprecated features by the CSM release in which they were deprecated, in reverse chronological order (the most recently deprecated
+features are listed first).
+
+## Deprecated in CSM 1.2
+
+* HMNFD v1 REST API has been deprecated as of CSM version 1.2. The v1 HMNFD APIs are targeted for removal in the CSM 1.5 release.
```

Comment:
Looking through the page, I see a bunch of acronyms used without ever being defined. CRUS, IMS, SCSD, UAS, UAI, etc. I think it may make more sense for me to make them all into links to our glossary page, to reduce clutter on this page.

Reply:
I made a quick pass at doing this. Some of the acronyms did not currently have entries in the glossary. If I have time, I'll add them and link them.